linker/shared/src/test/scala/org/scalajs/linker/EmitterTest.scala

```diff
@@ -53,7 +53,9 @@ class EmitterTest {
       |  */ $t // foo
     """.stripMargin.trim() + "\n"
 
-    val config = StandardConfig().withJSHeader(header)
+    val config = StandardConfig()
+      .withCheckIR(true)
```

Comment:
I think this should go inside `linkToContent`. It would follow the general pattern of adding `.withCheckIR(true)` in the most deeply nested method that directly manipulates the `linker`.